src/install_host_app.sh

```diff
@@ -126,17 +127,15 @@ echo "Installing $BROWSER_NAME host config"
 mkdir -p "$TARGET_DIR"
 
 PATH_ESC="$(echo "$PATH" | sed -e 's/@/\\@/g')"
-PASS_PATH_ESC="$(echo "$PASS_PATH" | sed -e 's/@/\\@/g')"
+PATH_HOMED="$(echo "$PATH_ESC" | sed -e "s@$HOME@~@g")"
 HOST_FILE_PATH_ESC="$(echo "$HOST_FILE_PATH" | sed -e 's/@/\\@/g')"
 PYTHON3_PATH_ESC="$(echo "$PYTHON3_PATH" | sed -e 's/@/\\@/g')"
 
 # Replace path to python3 executable \
-# Replace path to pass (only in a line starting with "COMMAND =") \
 # Set the PATH to match this script's \
 HOST_SED=" \
 1 s@.*@#!${PYTHON3_PATH_ESC}@; \
-/^COMMAND *=/s@\"pass\"@\"$PASS_PATH_ESC\"@; \
```

Comment:
Removing this line will break things for some users.

Reply:
Here is how I see it:

1. You use `$PATH` at install time to resolve `pass` to an absolute path;
2. You write `$PATH` at install time into the script;
3. The Python script uses `subprocess.Popen` with `env` including `$PATH`, thus the command will be searched in the `$PATH`;
4. Python finds the same `pass` as the install script and runs it.

The only breakage I can see is the case where the user moves `pass` to a directory not included in `$PATH`, but then, the hard-coded path to `pass` will also break.

Can you enlighten me how this might cause things to break for other users?

Reply:
Sorry that it took me so long to answer this. Here is what I found with `git blame`: https://github.com/passff/passff-host/pull/8

Reply:
There is a good reason why I avoid arcane Unices like the plague these days. MacOS is so bad under the hood actually. But alright, keep it as is for maximum reach. Hell will freeze over before MacOS actually becomes sensible.

Reply:
Thanks for your understanding :)

I agree, MacOS is a pain, and I wouldn't support it as a maintainer if people expected me to take care of the actual adaptation work. I will only "support" MacOS as a maintainer as long as the MacOS users in the community take care of the "actual" support themselves. Same with Windows.